<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientWidget</class>
 <widget class="QWidget" name="ClientWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>322</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MultiKinect TCPIP Controller</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>TCP/IP Client</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEdit_Port">
        <property name="text">
         <string>1234</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLineEdit" name="lineEdit_Message">
        <property name="text">
         <string>Hello World! :)</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="pushButton_Connect">
        <property name="text">
         <string>Connect</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QPlainTextEdit" name="plainTextEdit_sent"/>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="pushButton_Disconnect">
        <property name="text">
         <string>Disconnnect</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLineEdit" name="lineEdit_IP">
        <property name="text">
         <string>10.85.51.110</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QPushButton" name="pushButton_Send">
        <property name="text">
         <string>Send Message</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Port</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>IP</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QPlainTextEdit" name="plainTextEdit_received"/>
      </item>
      <item row="2" column="1" rowspan="2">
       <widget class="QLabel" name="label_ConnectionState">
        <property name="text">
         <string>TCP/IP State</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QPushButton" name="pushButton_Reboot">
        <property name="text">
         <string>Reboot</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QPushButton" name="pushButton_GitUpdate">
        <property name="text">
         <string>Git Update</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Kinect 2 Handler</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QPushButton" name="pushButton_Connect_Devices">
        <property name="text">
         <string>Connect All sensors</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="pushButton_Disconnect_Devices">
        <property name="text">
         <string>Disconnect All sensors</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="pushButton_Grab_Devices">
        <property name="text">
         <string>Grab All Data</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
